pkg/syndesis/operation/attach.go

```diff
@@ -52,47 +52,27 @@ func getAllManagerSelectors() []metav1.ListOptions {
 		{
 			LabelSelector: "app=syndesis,syndesis.io/app=todo",
 		},
-		// In the 7.0 template, the database does not have syndesis.io/type=infrastructure label
-		// It can be removed when it's no longer necessary to upgrade from 7.0 (together with the "legacy" package)
-		{
-			LabelSelector: "syndesis.io/app=syndesis,syndesis.io/component=syndesis-db",
```

Comment:
Is the label fixed in current 7.0 installations? Otherwise we should leave this.

Reply:
Yes, it's fixed and merged --> https://github.com/syndesisio/syndesis/pull/3163

going to move this PR over to Syndesis and lets merge it there.